src/index.html

```diff
@@ -1,13 +1,220 @@
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">home</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">about us</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#now-on-view" class="menu__link">now on view</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#explore-online" class="menu__link">explore online</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">contacts</a>
+            </li>
+          </ul>
+          <div class="menu__call menu__call--fifth-ave">
+            the met fifth avenue
+            <br>
+            <a
+              href="tel:+1 212-535-7710"
+              class="menu__call menu__call--phone"
+            >
+              tel:+1 212-535-7710
+            </a>
+          </div>
+
+          <div class="menu__call menu__call--cloisters">
+            The Met Cloisters
+            <br>
+            <a
+              href="tel:+1 212-923-3700"
+              class="menu__call"
+            >
+              tel:+1 212-923-3700
+            </a>
+          </div>
+          <label class="menu__call menu__call--action" for="#call-cbox">
+            call to order
+          </label>
+
+          <input
+            type="checkbox"
+            class="menu__call-cbox"
+            id="#call-cbox">
+
+          <div class="menu__call menu__call--addresses">
+            <a
+              href="tel:+1 212-535-7710"
+              class="menu__call-to-order"
+              >
+                fifth avenue
+            </a>
+
+            <a
+              href="tel:+1 212-923-3700"
+              class="menu__call-to-order"
+              >
+                cloisters
+            </a>
+          </div>
+        </div>
+      </div>
+      </div>
+    </nav>
+
+    <main>
+      <section class="page__section" id="about-us">
+        <div class="container">
+          <div class="grid">
+            <div class="grid__item grid__item--1-4">
+              <h2 class="grid__item grid__item-museum-hours">
+                Museum hours
+              </h2>
+            </div>
+            <div class="grid__item grid__item--6-11">
+              <div class="grid__item opening-hours">
+                <h3>
+                  The Met Fifth Avenue
+                </h3>
+                <span class="opening-hours opening-hours--hours">
+                  <div>10 am - 5 pm</div>
+                  <div>5 am - 10 pm</div>
+                  <div>closed</div>
+                </span>
+                <span class="opening-hours opening-hours--days">
+                  <div>Sun, Mon, Tue, Thu</div>
+                  <div>Fri, Sat</div>
+                  <div>Wen</div>
+                </span>
+                <div>
+                  <div class="opening__hours">
```

Comment:
I think you wrote it wrong. There isn't any block named 'opening'.  opening__hours makes it children of opening